pkg/kubelet/metrics/metrics.go

```diff
@@ -101,7 +122,7 @@ var (
 			Help:      "Interval in microseconds between relisting in PLEG.",
 		},
 	)
-	// Metrics of remote runtime operations.
+	// Metrics for remote runtime operations
```

Comment:
Mostly I see a slow convergence towards comments being complete sentences, with full stop at the end and so on.